components/LoadingSpinner/index.js

```diff
@@ -0,0 +1,9 @@
+import Circle from './style';
+
+export default function LoadingSpinner() {
+  return (
+    <Circle>
+      {Array.from({ length: 12 }).map((_, i) => <div key={i} className={`sk-child sk-circle${i + 2}`} />)}
+    </Circle>
```

Comment:
Hey Roland first of all there is no need to apologise to me. Thank you for your feedback which I am glad about.  I thought the spinner was pretty cool too and this is how I understood the function.  

Array.from({ length: 12 }) creates a new array with a length of 12 elements. each element is undefined as no second argument is given.

.map((_, i) => 
here the array method ,map is applied which applies a function to each element in the array.

<div key={i} className={sk-child sk-circle${i + 2}} /> here a div element with a unique key and two classes is created for each element. 

According to my understanding, 12 div elements are created, each with a unique CSS class. 

You're right that I didn't write the loading spinner myself. It could also be that I don't understand everything correctly. But it does work and I didn't just copy it blindly but tried to understand it. But if it's necessary, I just delete the loading spinner component and display a boring text instead.